src/components/HomePage/HomePage.jsx

```diff
@@ -1,23 +1,30 @@
 import React from 'react';
 import Container from 'react-bootstrap/Container';
 import { Row, Col } from 'react-bootstrap';
+import ConstructionBanner from '../ConstructionBanner/ConstructionBanner';
 import './HomePage.css';
 
 export default function HomePage() {
   return (
     <>
-      <Container fluid className="p1">
-        Be the voice of the change
-        <br />
-        you want to see around you.
-        <div className="p2">
-          <p />
-          Find how your elected officals have voted in relation with climate
-          change and
-          <u>make your voice heard.</u>
-        </div>
-      </Container>
-
+      {process.env.NODE_ENV === 'production' ? (
```

Comment:
The purpose of banner is to replace the whole text, as per design in Trello card. In your code, it just replaces the first part of the home page. It should be taken outside of the return statement and returned instead of the whole home page code.

Reply:
Oh! Sorry, we misunderstood that, we thought it was just the banner that was going to be switched. Will correct :)